project-management.md

```diff
@@ -36,8 +36,8 @@ All projects progress through a lifecycle. Projects are tracked on the [Project
 
 The project lifecycle is as follows:
 
-* A **Project Proposal** pull request is created, as described above.
-* If a project is approved, the pull request is merged. The project is then added to the list of **Potential Projects**. This list is roughly ordered based on priority.
+* A **Project Proposal** pull request is created, as described above. The pull request is then added to the **No Status** column on the Project Board.
+* For a project to be approved, its pull request must be approved by a majority of GC members. If a project is approved, and all merge requirements are met, the pull request is merged. The project is then transitioned to the list of **Potential Projects**. This list is roughly ordered based on priority.
```

Comment:
Do we need to vote on projects during GC calls, via approval of the PR or otherwise? I think being even more specific can help. Does majority mean we need 5 approvals or is it lazy voting at some point?

Reply:
this is non-blocking

Reply:
Sorry I didn't get to this... I think I'd prefer for it to be 5 approvals on the PR from GC members.